contrib/completion/bash/docker

```diff
@@ -3305,6 +3305,7 @@ _docker_service_update_and_create() {
 		--health-start-period
 		--health-timeout
 		--hostname
+		--isolation
```

Comment:
This option is Windows-specific. There is a way to selectively add the completion only when run against a daemon running on Windows, see https://github.com/docker/cli/blob/master/contrib/completion/bash/docker#L2456-L2458.

Reply:
It is not: Your swarm master can be running Linux, but your cluster might contain Windows nodes and thus accept services with isolation set.

Reply:
Ah yes, you're right.

Reply:
Ah, that keeps tripping me up as well 😅